app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command_line: str) -> None:
+    command, old_file, new_file = command_line.split(" ")
+    content = ""
+    if command == "cp":
```

Comment:
it must do nothing in case the user trying to copy file to file with the same name

Reply:
Fixed